src/test/java/com/adobe/epubcheck/ocf/OCFCheckerTest.java

```diff
@@ -394,24 +394,6 @@ public void testInvalidLoremForeign30()
   }
 
   @Test
-  public void testInvalidLoremMimetype30()
-  {
-    ValidationReport testReport = testOcfPackage("/30/expanded/invalid/lorem-mimetype/",
-        EPUBVersion.VERSION_3);
-    if (1 != testReport.getErrorCount() || 0 != testReport.getWarningCount())
-    {
-      outWriter.println(testReport);
-    }
-
-    List<MessageId> errors = new ArrayList<MessageId>();
-    Collections.addAll(errors, MessageId.PKG_007);
-    assertEquals(errors, testReport.getErrorIds());
-    assertEquals(0, testReport.getWarningCount());
-
-    assertTrue(testReport.hasInfoMessage("[format version] 3.0.1"));
-  }
-
-  @Test
```

Comment:
you're right, as I commented in commit log c6cae6daaa130ab892c73effcf1ef18fb85cdb56, `testValidateEPUBPLoremMimetype()` in src/test/java/com/adobe/epubcheck/api/Epub30CheckExpandedTest.java  do test against the same expanded EPUB folder (src/test/resources/30/expanded/invalid/lorem-mimetype/) and checck an error `MessageId.PKG_007`.

https://github.com/IDPF/epubcheck/blob/c1808ac4f7e2fcb61e0f30c204322278864964db/src/test/java/com/adobe/epubcheck/api/Epub30CheckExpandedTest.java#L101-L106